highfidelity/hifi
# Add placeholder image when previous snapshot can't be loaded

**Test Plan:**
Clear your settings. You can use desktop or HMD mode for this test plan.
1. Using the Snapshot app, choose a Snapshot location and take a "Still + GIF" snapshot.
2. When the GIF finishes processing, verify that both images appear in the Snapshot app, and that both images have the "Share" overlay appear on top of them.
3. Close the Snapshot app.
4. From your file explorer, delete the GIF snapshot from your local disk.
5. Open the Snapshot app.
6. Verify that the Still snapshot still appears and has the Share overlay appear on top of it.
7. Verify that the GIF snapshot no longer appears, and is instead replaced by a placeholder image that reads "Could not find last image taken". Also verify that the Share overlay does not appear on top of this placeholder image.
8. Take another set of Still + GIF snapshots.
9. When the GIF finishes processing, verify that both images appear in the Snapshot app, and that both images have the "Share" overlay appear on top of them.
